docs/UserGuide.md

```diff
@@ -138,6 +138,22 @@ Preference types: `high`, `low`, `recent`, `distant`
 * `recent`: Persons with more recent "last seen" dates rise to the top
 * `distant`: Persons with more distant "last seen" dates rise to the top
 
+### Save sort preference
```

Comment:
To stay consistent, you can consider adding the command word here. i.e. "Save sort preference: save_sort or svp"

Reply:
yep okay